datahub-web-react/src/app/entity/shared/containers/profile/header/PlatformContent/PlatformContentView.tsx

```diff
@@ -162,6 +181,13 @@ function PlatformContentView(props: Props) {
                 </ParentContainersWrapper>
                 {directParentContainer && <ContainerLink container={directParentContainer} />}
             </StyledTooltip>
+            {[...(parentNodes || [])]?.reverse()?.map((parentNode, idx) => (
```

Comment:
why do you need the extra [... ]  here? Also the ? 

(parentNodes || []).reverse

Reply:
reverse operates in place so we need the `...` to avoid weird state mutations